x-pack/legacy/plugins/security/server/lib/authorization/api_authorization.ts

```diff
@@ -12,8 +12,12 @@ export function initAPIAuthorization(server: Server, authorization: Authorizatio
   const { actions, checkPrivilegesDynamicallyWithRequest, mode } = authorization;
 
   server.ext('onPostAuth', async (request: Request, h: ResponseToolkit) => {
-    // if the api doesn't start with "/api/" or we aren't using RBAC for this request, just continue
-    if (!request.path.startsWith('/api/') || !mode.useRbacForRequest(request)) {
+    // if the api doesn't include or end with "/api/" or we aren't using RBAC for this request,
+    // just continue
+    if (
```

Comment:
__note:__ temp fix, we need to discuss what to do here with plugin-id prefixed NP API routes.

Reply:
I'm trying to remember why we had the `startsWith('/api/')` check to begin with...I don't think it's necessary for us to check for that path part in addition to the existence of the `access:foo` tag. We should support securing any route with an appropriately configured `access:` tag, even if it doesn't have `/api/` within its path.

This may have just been a holdover from an earlier implementation for securing API endpoints - @kobelb do you remember differently?

Reply:
Reverted this change since we can keep `/api/features` now.